perfzero/lib/cloud_manager.py

```diff
@@ -217,171 +248,146 @@ def list_all(project):
 
-  cmd = 'gcloud compute instances start {} --project={} --zone={}'.format(
-      instance_name, project, zone)
-  run_command(cmd, is_from_user=True)
-  logging.debug('\nSuccessfully started gcloud computing instance %s of '
-                'project %s in zone %s', instance_name, project, zone)
 
-
-def stop(username, project, zone):
-  instance_name = get_instance_name(username)
-  logging.debug('Stopping gcloud computing instance %s of project %s in '
-                'zone %s', instance_name, project, zone)
-
-  cmd = 'gcloud compute instances stop {} --project={} --zone={}'.format(
-      instance_name, project, zone)
+def stop(instance_name, project, zone, num_instances):
+  instance_names = _instance_names(instance_name, num_instances)
+  logging.debug('Stopping gcloud computing instance {} of project {} in '
+                'zone {}'.format(instance_name, project, zone))
+  cmd = _instances_cmd('stop', instance_names, project, zone)
   run_command(cmd, is_from_user=True)
-  logging.debug('\nSuccessfully stopped gcloud computing instance %s of '
-                'project %s in zone %s', instance_name, project, zone)
+  logging.info('Successfully stopped gcloud computing instances {} of '
+               'project {} in zone {}'.format(instance_names, project, zone))
 
 
-def delete(username, project, zone):
-  instance_name = get_instance_name(username)
-  logging.debug('Deleting gcloud computing instance %s of project %s in '
-                'zone %s', instance_name, project, zone)
-
-  cmd = 'echo Y | gcloud compute instances delete {} --project={} --zone={}'.format(  # pylint: disable=line-too-long
-      instance_name, project, zone)
+def delete(instance_name, project, zone, num_instances):
+  instance_names = _instance_names(instance_name, num_instances)
+  logging.debug('Deleting gcloud computing instance {} of project {} in '
+                'zone {}'.format(instance_name, project, zone))
+  cmd = 'echo Y | ' + _instances_cmd('delete', instance_names, project, zone)
   run_command(cmd, is_from_user=True)
-  logging.debug('\nSuccessfully deleted gcloud computing instance %s of '
-                'project %s in zone %s', instance_name, project, zone)
+  logging.info('Successfully deleted gcloud computing instances {} of '
+               'project {} in zone {}'.format(instance_names, project, zone))
 
 
-def parse_arguments(argv, command):  # pylint: disable=redefined-outer-name
+def parse_arguments():  # pylint: disable=redefined-outer-name
   """Parse command line arguments and return parsed flags.
 
-  Args:
-    argv: command line arguments
-    command: the subcommand requested by the user
-
   Returns:
     parsed flags
   """
-
-  # pylint: disable=redefined-outer-name
   parser = argparse.ArgumentParser(
-      usage='cloud_manager.py {} [<args>]'.format(command),
-      formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+      epilog='''The supported commands are:
```

Comment:
It seems that the helper message is a bit confusing after this change. For example, `python perfzero/lib/cloud_manager.py status -h` will print arguments such as `--accelerator_type` that does not below to this command.

It would make sense to update to helper message from `cloud_manager.py -h` to make it clear that user can run e.g. `cloud_manager.py status -h` to get specific helper message for each command.